src/index.js

```diff
@@ -26,11 +32,13 @@ $(document).ready(() => {
 $('.main-button__submit-name').on('click', (e) => {
   $('.splash').hide();
   $('.board').show();
+  game = new Game(data);
```

Comment:
Since you’re waiting for a click event to instantiate your new Game with all of that data, you probably won’t run into any timing issues with getting that fetch request back in time, but there’s still potential for that to happen. You might want to do this instantiation/set up this click handler *inside* of that `.then()` so that you know it can’t happen until the data has come back.